auctions/english-auction/src/events.sw

```diff
@@ -0,0 +1,28 @@
+library events;
+
+dep data_structures;
+
+use data_structures::{Asset, Auction};
+use std::identity::Identity;
+
+pub struct AuctionCancelEvent {
+    auction: Auction,
+    auction_id: u64,
+}
+
+pub struct AuctionStartEvent {
+    auction: Auction,
```

Comment:
@matt-user has recently asked a similar question however for now I think it's better to include the creator directly in the event and if it turns out that we can filter inside the Auction then that can be removed later